src/components/sbb-header-action/sbb-header-action.scss

```diff
@@ -46,6 +46,7 @@ $breakpoints: 'zero', 'micro', 'small', 'medium', 'large', 'wide', 'ultra';
 @include sbb.hover-mq($hover: true) {
   :host(:hover) {
     --sbb-header-action-background-color: var(--sbb-color-milk-default);
+    --sbb-header-action-translate: #{sbb.px-to-rem-build(-1)};
```

Comment:
I know it's not necessary, but I'd suggest to initialize --sbb-header-action-translate-y with 0 on :host{}.